tests/fixed/array_indentation.php

```diff
@@ -22,3 +22,7 @@
         9,
     ],
 ];
+
+MyClass::baz([
+    new MyObject(),
+]);
```

Comment:
Report back to https://github.com/squizlabs/PHP_CodeSniffer/issues/2939 this case, but let's keep it like this so when the bugfix arrives, we'll know 😉

Reply:
If this wasn't introduced with the patch version and breaks stuff on our side, I agree.